src/main/java/website/frontrow/artificial/intelligence/ArtificialIntelligence.java

```diff
@@ -184,4 +189,29 @@ public void addToLog(String action)
     {
         Log.add(action);
     }
+    
+    /**
+     * Assigns the enemies to the second player if the first dies.
+     */
+    public void nextPlayer()
+    {
+    	if(players.size() > 1)
+		{
+			this.player = this.players.get(1);
```

Comment:
You hardcoded a second player in here. 
This should work fine for a two player game, as we have at the moment. 
But this will break again when we want a third or a fourth player.

Just a small concern.
< interface